lua/mLRS.lua

```diff
@@ -1374,7 +1374,7 @@ local function scriptRun(event)
         return 2
     end
     if mbridge == nil or not mbridge.enabled() then
-        if model.getModule(1).Type ~= 5 then
+        if model.getModule(0).Type ~= 5 and model.getModule(0).Type ~= 5 then
```

Comment:
this looks like a bug, right? one should be (1), shouldn'tt it

Reply:
Yup, JLP noticed it didn't work on external modules and I fixed it.  Thanks!